### PR TITLE
update packages

### PR DESCRIPTION
**Pull request by PieterGit**
@mddub 
This package has quite some outdated packages and some security issues.

```
$ npm install
npm WARN deprecated jade@0.26.3: Jade has been renamed to pug, please install the latest version of pug instead of jade
npm WARN deprecated minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated graceful-fs@2.0.3: please upgrade to graceful-fs 4 for compatibility with current and future versions of Node.js
added 19 packages from 246 contributors and audited 87 packages in 4.445s
found 10 vulnerabilities (2 low, 6 moderate, 1 high, 1 critical)
  run `npm audit fix` to fix them, or `npm audit` for details
```

This is a partial fix for https://github.com/mddub/minimed-connect-to-nightscout/issues/6

Only problem I have is with getting `test/transform.js` ok and I can't test this package because I don't use minimed. 

The following test does not yet work:

````
  it('should discard data more than 20 minutes old', function() {
    var now = Date.now();
    var THRESHOLD = 20;
    var boundary = now - THRESHOLD * 60 * 1000;
    expect(
      transform(
        f.data({'currentServerTime': now, 'lastMedicalDeviceDataUpdateServerTime': boundary})
      ).entries.length
    ).to.be.greaterThan(0);

```

It errors with:

```
RangeError: Invalid time value
    at Date.toISOString (<anonymous>)
    at timestampAsString (C:\Users\pietergit\Documents\src\minimed-connect-to-nightscout\transform.js:23:30)
    at deviceStatusEntry (C:\Users\pietergit\Documents\src\minimed-connect-to-nightscout\transform.js:66:16)
    at module.exports (C:\Users\pietergit\Documents\src\minimed-connect-to-nightscout\transform.js:129:20)
    at Context.<anonymous> (C:\Users\pietergit\Documents\src\minimed-connect-to-nightscout\test\transform.js:37:7)

```

Can you or come other minimed-connect-to-nightscout user help complete this PR?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,12 +21,13 @@
   },
   "dependencies": {
     "common": "0.2.x",
-    "lodash": "3.10.x",
-    "request": "2.64.x"
+    "lodash": "^4.17.10",
+    "pug": "^2.0.3",
```
Comment (mddub) [inline package.json]: You don't need this line: `pug` is a dependency of `mocha`, so the update to `mocha ^5.2.0` below takes care of the `npm install` warning about jade/pug.

**Review by mddub — CHANGES_REQUESTED** [review]
This looks great! Thanks for doing this! One small change.

Once this change is made and `npm-shrinkwrap.json` looks good, I'll merge, release a 1.1.1, and publish to npm.

**Comment by mddub** [comment]
And thanks for calling attention to the failing test - I fixed it in #8.

File: package.json (hunk 1 above)
Comment (PieterGit) [reply package.json]: fixed

**Comment by PieterGit** [comment]
@mddub thanks for fixing the test. I did the changes you suggested. Can you please publish it to npm as a 1.1.1 version, and I will include it in the 0.10.3 Nightscout release.

**Review by mddub — APPROVED** [review]

**Comment by mddub** [comment]
@PieterGit Thanks, I've published v1.1.1 to npm: https://www.npmjs.com/package/minimed-connect-to-nightscout

**Comment by PieterGit** [comment]
@mddub thanks for making the release. I integrated it into Nightscout with this PR
https://github.com/nightscout/cgm-remote-monitor/pull/3701

This will be released with Nightscout 0.10.3

**Pull request merged**